README.md

```diff
@@ -304,24 +305,24 @@ Please note that the default for bracket is `square` defined by the [attribute](
 
 #### Lowercase only ASCII characters in auto link ids
 
-The plugin lowercase all alphabets within auto link ids default.
+The plugin lowercase **only ascii alphabets**(`a` to `z`) within auto link ids default.
 
 ```
-<!-- MarkdownTOC autolink=true -->
+<!-- MarkdownTOC autolink="true" -->
 
-- [ПРИМЕР EXAMPLE][пример-example]
+- [ПРИМЕР EXAMPLE][ПРИМЕР-example]
 
 <!-- /MarkdownTOC -->
 
 # ПРИМЕР EXAMPLE
 ```
 
-But you can also squeeze its target **only ascii alphabets**(`a` to `z`) with `lowercase_only_ascii=true` attribute.
+But you can also expand its target all alphabets with `lowercase_only_ascii="false"` attribute.
```

Comment:
Rephrase: "You can expand the lowercasing capability by setting the `lowecase_only_ascii` attribute to `false`"